setup.py

```diff
@@ -22,6 +22,7 @@
     "pandas",
     "requests<2.30.0",
     "fsspec>=2023.5.0",
+    "vellum-ai>=0.0.14",
```

Comment:
nit: let's have user lazily install `vellum-ai` instead of adding it as a direct dependency. This is how we've been treating third-party clients/libraries (e.g. vector DBs)

Reply:
Can just throw import error when attempting to use vellum predictor without having `vellum-ai` lib installed.